lab/webapp/src/components/Dataset/components/DatasetMenu/components/Charts/BarCharts/barChart.test.js

```diff
@@ -0,0 +1,42 @@
+import BarCharts from './';
```

Comment:
May want to rename the "BarCharts" component to something along the lines of "StackedBarChart" for clarity